backend/src/routes/v0/auth.py

```
@@ -144,40 +110,45 @@
-	access_token = create_access_token(user)
-	# Update airtable with latest login
-	try:
+		# Create access token with full user object
+		access_token = create_access_token(user)
+		# Update airtable with latest login
+	
 		await airtable_service.latest_login(user.email)
 	except Exception as e:
-		await airtable_service.create_contact(user_response)
+		logger.exception(f"Error logging in: {e}")
+		raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)